src/server/server_family.cc

```diff
@@ -491,20 +491,19 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
   if (ReplicaOfFlag flag = GetFlag(FLAGS_replicaof); flag.has_value()) {
     service_.proactor_pool().GetNextProactor()->Await(
```

Comment:
I'd add a WARNING print, in case `--dbfilename` *is* set, in addition to `--replicaof`

Reply:
but dbfilename can be also set for saveing which we should be able to do when running as replica

Reply:
Ok, that makes sense. I thought that warning saying that we're not loading the RDB may be helpful, but you're right that it should be a common practice. Please ignore.